### PR TITLE
Add instructions to use Testcontainers Desktop

### PR DESCRIPTION
**Pull request by sivaprasadreddy**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,14 +16,19 @@
 
 	<properties>
 		<java.version>17</java.version>
+		<testcontainers.version>1.19.0</testcontainers.version>
```
Comment (eddumelendez) [inline pom.xml:19]:
```suggestion
		<testcontainers.version>1.19.1</testcontainers.version>
```

File: pom.xml [hunk 2]
```diff
@@ -16,14 +16,19 @@
...
 	</properties>
 
     <dependencyManagement>
         <dependencies>
+			<!--
+				You don't need to configure testcontainers-bom
+				if you are using Spring Boot 3.1.0+
```
Comment (eddumelendez) [inline pom.xml:26]: only if the Testcontainers latest version doesn't match with the one provided by SB

File: step-1-getting-started.md [hunk 3]
```diff
@@ -2,15 +2,22 @@
...
+Testcontainers Desktop is a free tool that can help you during the local development and debugging of your tests.
+You can download the app from [https://testcontainers.com/desktop/](https://testcontainers.com/desktop/).
 
 ## Check Docker
 
-Make sure you have a Docker environment available on your machine. 
+To use Testcontainers, you need to have a supported Docker environment.
+If you have the Testcontainers Desktop application installed, it will show you the available container runtimes on your machine.
 
+* You can choose any of the installed container runtimes like **Docker Desktop**, **OrbStack**, **Rancher Desktop**, **Podman**, etc.
```
Comment (eddumelendez) [inline step-1-getting-started.md:16]: Podman is not displayed

File: step-1-getting-started.md [hunk 4]
```diff
@@ -2,15 +2,22 @@
...
 * It can be [Testcontainers Cloud](https://testcontainers.com/cloud) recommended to avoid straining the conference network by pulling heavy Docker images. 
+* If you don't have any container runtimes installed on your machine, you can also use the **Embedded Runtime** provided by Testcontainers Desktop.
```
Comment (eddumelendez) [inline step-1-getting-started.md:18]: should we add the config property to enable it?

Comment (sivaprasadreddy) [reply step-1-getting-started.md:18]: Added this info.

File: step-1-getting-started.md (hunk 3 above)
Comment (sivaprasadreddy) [reply step-1-getting-started.md]: Removed Podman from the list.

**Review by eddumelendez — APPROVED** [review]

File: pom.xml [hunk 5]
```diff
@@ -16,20 +16,10 @@
...
 		<java.version>17</java.version>
+		<!-- If you want to use a different version of Testcontainers -->
+		<testcontainers.version>1.19.1</testcontainers.version>
 	</properties>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.testcontainers</groupId>
-                <artifactId>testcontainers-bom</artifactId>
```
Comment (shelajev) [inline pom.xml:25]: Did we get rid of the bom?

**Review by kiview — APPROVED** [review]

File: pom.xml (hunk 5 above)
Comment (kiview) [reply pom.xml:25]: Is this Spring-managed now?

File: step-1-getting-started.md [hunk 6]
```diff
@@ -2,15 +2,23 @@
...
+Testcontainers Desktop is a free tool that can help you during the local development and debugging of your tests.
+You can download the app from [https://testcontainers.com/desktop/](https://testcontainers.com/desktop/).
 
 ## Check Docker
 
-Make sure you have a Docker environment available on your machine. 
+To use Testcontainers, you need to have a supported Docker environment.
+If you have the Testcontainers Desktop application installed, it will show you the available container runtimes on your machine.
 
+* You can choose any of the installed container runtimes like **Docker Desktop**, **OrbStack**, **Rancher Desktop**, etc.
```
Comment (kiview) [inline step-1-getting-started.md:16]: @sivaprasadreddy Do you know if Rancher Desktop works throughout the workshop without any issues?

File: pom.xml (hunk 5 above)
Comment (sivaprasadreddy) [reply pom.xml:25]: Yes, from Spring Boot 3.1.0 they have configured testcontainers-bom in parent pom.

File: step-1-getting-started.md (hunk 6 above)
Comment (sivaprasadreddy) [reply step-1-getting-started.md:16]: Tried running workshop with TCD + Rancher Desktop and it worked well for me.

**Comment by kiview** [comment]
Thanks @sivaprasadreddy, I merged the PR to have it ready for todays workshop 🙂

**Pull request merged**